tests/integration/docker/swarm.bats

```diff
@@ -28,22 +28,45 @@ url=http://127.0.0.1:8080/hostname
 # number of attemps to obtain the hostname
 # of the replicas using curl
 number_of_attemps=5
+# saves the hostname of the replicas
 declare -a REPLICAS
+# saves the process id of the replicas
+declare -a REPLICAS_UP
+# saves the name of the replicas
+declare -a NAMES_IPS_REPLICAS
+# saves the ip of the replicas
+declare -a IPS_REPLICAS
+
 
 setup() {
 	source $SRC/test-common.bash
 	clean_docker_ps
 	runtime_docker
 	$DOCKER_EXE swarm init
-}
-
-@test "check that the replicas' names are different" {
 	# currently using mcastelino/nginx but it will be
 	# modified when nginx image is working
 	$DOCKER_EXE service create --name testswarm --replicas $number_of_replicas --publish 8080:80 mcastelino/nginx /bin/bash -c "hostname > /usr/share/nginx/html/hostname; nginx -g \"daemon off;\"" 2> /dev/null
-	while [ `docker ps -a | tail -n +2 | egrep "Up [0-9]*"|  wc -l`  -lt  $number_of_replicas ]; do
+	while [ `$DOCKER_EXE ps -a | tail -n +2 | egrep "Up [0-9]*"|  wc -l`  -lt  $number_of_replicas ]; do
```

Comment:
@mcastelino thank you very much for all the feedback, these commands are great because it makes everything easier :)